setup.py

```diff
@@ -54,6 +54,7 @@
     include_package_data=True,
     install_requires=[
         "eth-abi>=2.0.0b4,<3.0.0",
+        "eth-account==0.5.6",
```

Comment:
Let's open this up to be non-breaking changes. Since it's one the snake-charmers maintain, I'm relatively confident it won't break unless be bump the minor version.
```suggestion
        "eth-account>=0.5.6,<0.6.0",
```
It will help to play nice with our other libraries.